hstest/testing/plotting/seaborn_handler.py

```diff
@@ -0,0 +1,203 @@
+from .drawing import Drawing
+from importlib import reload
+from hstest.testing.plotting.matplotlib_handler import MatplotlibHandler
+
+
+class SeabornHandler:
+    _saved = False
+    _replaced = False
+
+    _displot = None
+    _histplot = None
+    _lineplot = None
+    _lmplot = None
+    _scatterplot = None
+    _catplot = None
+    _barplot = None
+    _violinplot = None
+    _heatmap = None
+    _boxplot = None
+
+    @staticmethod
+    def replace_plots(drawings):
+        try:
+            import seaborn as sns
+        except ModuleNotFoundError:
+            return
+
+        lib_type = 'seaborn'
+
+        def displot(data=None, **kwargs):
+            drawing = Drawing(
+                lib_type,
+                'displot',
+                {
+                    'data': data,
+                    'kwargs': kwargs
+                }
+            )
+            drawings.append(drawing)
+
+        def histplot(data=None, **kwargs):
+            drawing = Drawing(
+                lib_type,
+                'histplot',
```

Comment:
Why not `hist` like in Matplotlib and Pandas? If it is the same plot type (just with a different library) they should have the same name.

Reply:
The same question for `lineplot`, `lmplot`, `catplot`